helm/intergration/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for content-store.
```

Comment:
I can see from your description about where these files belong. My suggestion is that they don't belong in this repo as there's nothing to test them or any interaction with the app code.

I imagine that if we need to change something across lots of the helm files (such as change a hostname) it'd be a much simpler job for one repo with them all compared to PRs in each repo.